components/landing/site-counter.tsx

```diff
@@ -3,15 +3,20 @@ import {
     directDB_getSubmitCount,
     directDB_incrementSiteCount,
 } from "@/client-side-queries/direct-db-query/landing-page-queries";
+import { Box } from "../ui/stat-box";
 
 const SiteCounter = async () => {
     const siteCountValue = await directDB_getSiteCount();
     await directDB_incrementSiteCount(siteCountValue[1]!, siteCountValue[0]!);
     const landingPageSubmitCount = await directDB_getSubmitCount();
+
     return (
-        <article className='flex h-screen flex-col items-center justify-center text-center'>
-            <p>Site counter: {siteCountValue[0]}</p>
-            <p>Landingpage_submit_count: {landingPageSubmitCount}</p>
+        <article className='grid h-screen gap-4 sm:grid-cols-1'>
```

Comment:
It's okay in the sense that it looks like it works. But make the boxes smaller, and make them horizontal on large screens and vertical only on small screens. Avoid using grid because that could potentially mess things up